src/engine/HasPredicateScan.cpp

```diff
@@ -268,7 +272,7 @@ Result HasPredicateScan::computeResult([[maybe_unused]] bool requestLaziness) {
           TripleComponent::Iri::fromIriref(HAS_PATTERN_PREDICATE), std::nullopt,
           std::nullopt}
           .toScanSpecification(index);
-  const auto& perm = index.getPermutation(Permutation::Enum::PSO);
+  const auto& perm = permutation();
```

Comment:
It still seems inconsistent, that once you use a `triple` and once directly the `scan specification`, on which you can't do the `getPermutationFor...`